tests/chapter4-exercise-paragraphs-tests.md

```diff
@@ -48,3 +53,25 @@ Any tags that are open when encountering the second paragraph should be closed
          <p>
            <b>
              'world'
+			 
+			 
+Test when <p> tags have different attributes from each other			 
+	>>> test_parse("<p class='first-paragraph'>This has a class</p><p id='second-paragraph'>This has an ID</p>")
```

Comment:
I don't think this exercise triggers the "nested `<p>` tags" thing literally at all, since these `<p>` tags aren't nested.